app/models/mixins/custom_attribute_mixin.rb

```diff
@@ -27,6 +27,16 @@ module CustomAttributeMixin
     def self.custom_keys
       CustomAttribute.where(:resource_type => base_class).map(&:name).uniq
     end
+
+    def self.add_custom_attribute(custom_attribute)
+      virtual_column(custom_attribute.to_sym, :type => :string)
+
+      define_method(custom_attribute.to_sym) do
+        attr = custom_attribute.sub(CUSTOM_ATTRIBUTES_PREFIX, "")
+        value = miq_custom_attributes.or(ems_custom_attributes).find_by('lower(name) = ?', attr).try(:value)
```

Comment:
Oh I am sorry this was my old code and I have replaced it by code as you commented https://github.com/ManageIQ/manageiq/pull/9451#discussion-diff-69226923 - so moving here discussion.